### ISSUE TITLE
no vok file for simple files

### ISSUE THREAD

**Issue by hendriktews** [coq/coq#15773]
#### Description of the problem

For simple files, e.g., only containing
```
Definition a : nat := 4.
```
`coqc -vok` does not create a `.vok` file. (Of course after running `coqc -vos`.) Even if I add simple proofs such as
```
Lemma a_equal_4 : a = 2 * 2.
Proof using.
  simpl.
  trivial.
Qed.
```
no vok file is created. `.glob` files are still created.

I found this problem while investigating a CI failure in Proof General, see ProofGeneral/PG#632.

#### Coq Version

The Coq Proof Assistant, version 8.15.0
compiled with OCaml 4.13.1

**Comment by SkySkimmer** [comment]
fixed in https://github.com/coq/coq/pull/15745 (not yet merged)

**Comment by hendriktews** [comment]
The error does not depend on the content of the file - coq 8.15.0 never creates .vok files.

**Comment by RalfJung** [comment]
> fixed in #15745 (not yet merged)

It's merged now, so should the issue be closed?
I'm switching to the 8.15 branch for now since Coq is unusable for me when vok builds are not cached.
